chore: update registration form,  form validations, mobile menu, and style.css

danujamayadunne opened the pull request:
This PR introduce select option for food preference, mobile responsive menu, and a css change.

- Update registrationForm.tsx
- Add form validations
- Update page.tsx for mobile menu
- Increase width of menu on desktop

danujamayadunne committed:
chore: updated food preferences in registration form

danujamayadunne committed:
style: changed the width of meenu

danujamayadunne committed:
feat: added responsive menu for mobile

nethmalds reviewed (changes requested):
Overall nice... Align the menu to left hand side in mobile and tab view..

nethmalds committed:
Merge branch 'release/development' of https://github.com/hackathon-hub-nsbm/algoxplore-website into release/development

nethmalds committed:
Merge branch 'release/development' into feat/registerform

danujamayadunne committed:
chore: changed moile menu position for mobile

danujamayadunne committed:
chore: add form validations

nethmalds committed:
Merge branch 'release/development' of https://github.com/hackathon-hub-nsbm/algoxplore-website into release/development

danujamayadunne committed:
chore: add glow to register button

nethmalds committed:
Merge branch 'release/development' into feat/registerform

nethmalds committed:
style: update global CSS and main page styling
- Modified `globals.css` to improve styling consistency across the app.
- Updated `page.tsx` to enhance layout and visual elements on the main page.
